sql/core/src/test/resources/sql-tests/inputs/predicate-functions.sql

```diff
@@ -0,0 +1,42 @@
+-- In
+select 1 in(1, 2, 3);
+select 1 in(2, 3, 4);
+select named_struct('a', 1, 'b', 2) in(named_struct('a', 1, 'b', 1), named_struct('a', 1, 'b', 3));
+select named_struct('a', 1, 'b', 2) in(named_struct('a', 1, 'b', 2), named_struct('a', 1, 'b', 3));
```

Comment:
For `In`, I've complemented its test coverage in `PredicateSuite` and use `checkEvaluation`. I'd prefer to have it in a following PR if you don't insist to add it here.

Reply:
Looks like the existing test for `In` is not well covered. We should improve it. I will commit the tests I added locally.